gh_actions_exporter/metrics.py

```diff
@@ -216,7 +216,7 @@ def handle_job_duration(self, webhook: WebHook, settings: Settings):
             duration = (webhook.workflow_job.completed_at.timestamp()
                         - webhook.workflow_job.started_at.timestamp())
             self.job_duration.labels(**labels).observe(duration)
-        elif webhook.workflow_job.status == "in_progress":
-            duration = (webhook.workflow_job.steps[0].started_at.timestamp()
-                        - webhook.workflow_job.started_at.timestamp())
-            self.job_start_duration.labels(**labels).observe(duration)
+
+            start_duration = (webhook.workflow_job.steps[0].completed_at.timestamp()
+                              - webhook.workflow_job.started_at.timestamp())
+            self.job_start_duration.labels(**labels).observe(start_duration)
```

Comment:
Honestly i don't get who this could fix our probleme.
The `step[0].completed_at` depend of  the first step you run and could create fake data.
And if there is no real difference between `step[0].started_at`  and `workflow_job.started_at` the only thin you will calculate here is the duration of your first step